src/badguy/dart.cpp

```diff
@@ -39,13 +39,14 @@ Dart::Dart(const ReaderMapping& reader) :
   SoundManager::current()->preload("sounds/stomp.wav");
 }
 
-Dart::Dart(const Vector& pos, Direction d, const BadGuy* parent_ = nullptr) :
-  BadGuy(pos, d, "images/creatures/dart/dart.sprite"),
+Dart::Dart(const Vector& pos, Direction d, const BadGuy* parent_ = nullptr, const std::string& sprite, Flip flip) :
```

Comment:
The default parameter value of `parent_` is unneeded in .cpp.

Reply:
I'm suprised that it compiled lol. Default values in .cpp always caused compilation errors for me.